### PR TITLE
[SponsorBlock]: Support for Filler category

### PR DESCRIPTION
**Pull request by nihil-admirari**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Filler Tangent category has been recently added to SponsorBlock: https://github.com/ajayyy/SponsorBlockServer/commit/db48953e39a6874774666100c5c4b29ce31bf81b

### CODE REVIEW

**Comment by pukkandan** [comment]
I have also added the `poi_highlight` category and changed how the defaults are handled. Have a look

File: README.md [hunk 1]
```diff
@@ -937,19 +937,20 @@ Make chapter entries for, or remove various segments (sponsor,
...
+                                     interaction, music_offtopic, poi_highlight.
+                                     You can prefix the category with a "-" to
+                                     exempt it. See [1] for description of the
+                                     categories. Eg: --sponsorblock-mark all,-preview
+                                     [1] https://wiki.sponsor.ajay.app/w/Segment_Categories
     --sponsorblock-remove CATS       SponsorBlock categories to be removed from
                                      the video file, separated by commas. If a
                                      category is present in both mark and
                                      remove, remove takes precedence. The syntax
                                      and available categories are the same as
```
Comment (nihil-admirari) [inline README.md:951]: `poi_highlight` should not be documented as available for removal: it is always filtered out from `_remove_sponsor_segments`.

File: yt_dlp/options.py [hunk 2]
```diff
@@ -1434,20 +1434,28 @@ def _dict_from_options_callback(
...
+            f'Available categories are all, default(=all), {", ".join(SponsorBlockPP.CATEGORIES.keys())}. '
+            'You can prefix the category with a "-" to exempt it. See [1] for description of the categories. '
+            'Eg: --sponsorblock-mark all,-preview [1] https://wiki.sponsor.ajay.app/w/Segment_Categories'))
     sponsorblock.add_option(
         '--sponsorblock-remove', metavar='CATS',
         dest='sponsorblock_remove', default=set(), action='callback', type='str',
-        callback=_set_from_options_callback, callback_kwargs={'allowed_values': SponsorBlockPP.CATEGORIES.keys()},
-        help=(
+        callback=_set_from_options_callback, callback_kwargs={
+            'allowed_values': SponsorBlockPP.CATEGORIES.keys(),
```
Comment (nihil-admirari) [inline yt_dlp/options.py:1449]: `poi_highlight` should not be allowed here, see the comment above.

File: yt_dlp/postprocessor/sponsorblock.py [hunk 3]
```diff
@@ -10,18 +10,23 @@
 
 
 class SponsorBlockPP(FFmpegPostProcessor):
-
+    # https://wiki.sponsor.ajay.app/w/Types
     EXTRACTORS = {
         'Youtube': 'YouTube',
     }
+    POI_CATEGORIES = {
+        'poi_highlight': 'Highlight',
```
Comment (nihil-admirari) [inline yt_dlp/postprocessor/sponsorblock.py:18]: Are there gonna be multiple POI categories?

File: README.md (hunk 1 above)
Comment (pukkandan) [reply README.md:951]: True. The documentation is getting more and more complex though. So u have any ideas to clean it up/ simplify it?

File: yt_dlp/postprocessor/sponsorblock.py (hunk 3 above)
Comment (pukkandan) [reply yt_dlp/postprocessor/sponsorblock.py:18]: There are no plans for any other so far afaik. But having it defined separately makes it easier for us if one is added

File: README.md (hunk 1 above)
Comment (nihil-admirari) [reply README.md:951]: No, unfortunately.

**Pull request merged**